src/main/kotlin/no/nav/familie/baks/mottak/task/OpprettSøknadBehandlingISakTask.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.familie.baks.mottak.task
+
+import no.nav.familie.baks.mottak.integrasjoner.BehandlingStatus
+import no.nav.familie.baks.mottak.integrasjoner.BehandlingType
+import no.nav.familie.baks.mottak.integrasjoner.FagsakStatus
+import no.nav.familie.baks.mottak.integrasjoner.JournalpostClient
+import no.nav.familie.baks.mottak.integrasjoner.KontantstøtteOppgaveMapper
+import no.nav.familie.baks.mottak.integrasjoner.KsSakClient
+import no.nav.familie.baks.mottak.integrasjoner.RestMinimalFagsak
+import no.nav.familie.baks.mottak.integrasjoner.finnesÅpenBehandlingIFagsak
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = OpprettSøknadBehandlingISakTask.TASK_STEP_TYPE,
+    beskrivelse = "Oppretter behandling i sak etter automatisk journalføring av søknad",
+)
+class OpprettSøknadBehandlingISakTask(
+    private val journalpostClient: JournalpostClient,
+    private val kontantstøtteOppgaveMapper: KontantstøtteOppgaveMapper,
+    private val ksSakClient: KsSakClient,
+) : AsyncTaskStep {
+    val log: Logger = LoggerFactory.getLogger(OpprettSøknadBehandlingISakTask::class.java)
+
+    override fun doTask(task: Task) {
+        val journalpost = journalpostClient.hentJournalpost(task.metadata["journalpostId"] as String)
+        val fagsakId = task.metadata["fagsakId"] as String
+        val brukersIdent = task.metadata["personIdent"] as String
+        val tema = Tema.valueOf(journalpost.tema!!)
+
+        when (tema) {
```

Comment:
Tema.valueOf(journalpost.tema!!) kan flyttes inn i when

Reply:
```suggestion
        when (val tema = Tema.valueOf(journalpost.tema!!)) {
```

Sånn her tenkte du på?
Synes ikke det ble så fint, og siden vi referer til tema i senere log setning så må vi definere den 🤔 (og derfor må kjøre med val)